README.md

```diff
@@ -24,30 +34,33 @@ see https://neo4j.com/docs/operations-manual/current/backup/
 
 When working localy, docker-compose.override.yml adds a local bind mount to the /backups folder.
 
+The Crunchbase backup can be downloaded from Nexus. Decompress it in the ./backups bind mounted folder.
 
 Create an online backup:
 ```
-neo4j-admin backup --backup-dir /backups --database crunchbase-1.0.0
+docker-compose exec neo4j neo4j-admin backup --backup-dir /backups --database crunchbase-1.0.0
```

Comment:
I usualy exec into a container, and then run the list of commands, rather than exec every time.
Why : in case of failure, you need to iterate and access debug log

Reply:
OK, that's why you didn't prepended the `docker-compose exec`. I rather directly exec the commands from the outside, but this is a matter of taste. I'll revert theses changes and add a comment explicitly indicating that the commands have to be executed in the container.